### PR TITLE
VADC-1516: Updated colors for the buttons in the GWAS modal according to 508

### PR DESCRIPTION
**Pull request by vzpgb**
Link to JIRA ticket if there is one: [VADC-1516](https://ctds-planx.atlassian.net/browse/VADC-1516)

**Before**
image

**After**
image

### Bug Fixes
* Updated colors for the buttons in the GWAS modal according to 508
* Replaced "border" style with "border-color" style to affect colors only
* Updates Team Modal text color

[VADC-1516]: https://ctds-planx.atlassian.net/browse/VADC-1516?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Analysis/SharedUtils/AccessibilityUtils/AccessibilityCSS/Accessibility.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border: 0;
+}
+
+.ant-modal-content button:focus {
+  outline: 1px dashed var(--focus-outline-color);
+  outline-offset: 1px;
+}
```
Comment (jarvisraymond-uchicago) [inline src/Analysis/SharedUtils/AccessibilityUtils/AccessibilityCSS/Accessibility.css:27]: Please add a final line break here and in the other CSS files. Github adds a red warning icon to files missing this. 

Here's the reason why: https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b

**Review by jarvisraymond-uchicago — CHANGES_REQUESTED** [review]
Looks good, but please add final new lines to the files missing them.

File: src/Analysis/SharedUtils/AccessibilityUtils/AccessibilityCSS/Accessibility.css (hunk 1 above)
Comment (vzpgb) [reply src/Analysis/SharedUtils/AccessibilityUtils/AccessibilityCSS/Accessibility.css]: Thank you! Files updated @jarvisraymond-uchicago

**Review by jarvisraymond-uchicago — APPROVED** [review]
Looks good to me.

**Review by cemile-ctds — APPROVED** [review]
Looks good on my side.

**Pull request merged**